src/FakeItEasy.dotnet/FakeItEasy.nuspec

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>FakeItEasy</id>
+    <version>0.0.0</version>
+    <title>FakeItEasy - Mocking made faking easy!</title>
+    <authors>Patrik Hägne, FakeItEasy contributors</authors>
+    <description>It's faking amazing! The easy mocking framework for .NET that works great in C# and VB.NET alike. No need to know the difference between a stub, a mock or a spy, everything's a fake! The easy to use, refactoring friendly API makes faking a breeze.</description>
+    <releaseNotes>https://github.com/FakeItEasy/FakeItEasy/releases</releaseNotes>
+    <language>en-US</language>
+    <projectUrl>http://fakeiteasy.github.io/</projectUrl>
+    <iconUrl>http://fakeiteasy.github.io/img/fakeiteasy_logo_256_square_white.png</iconUrl>
+    <licenseUrl>https://github.com/FakeItEasy/FakeItEasy/blob/master/License.txt</licenseUrl>
+    <copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</copyright>
+    <tags>TDD unittesting mocks mocking fakes faking stubs stubbing spy spies doubles isolation substitutes substitution</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.0" />
+      <group targetFramework=".NETStandard1.6">
+        <dependency id="Microsoft.Extensions.DependencyModel" version="1.0.0" />
+        <dependency id="System.Collections.Concurrent" version="4.0.12" />
+        <dependency id="System.Console" version="4.0.0" />
+        <dependency id="System.Diagnostics.Tools" version="4.0.1" />
+        <dependency id="System.Diagnostics.Debug" version="4.0.11" />
+        <dependency id="System.IO.FileSystem" version="4.0.1" />
+        <dependency id="System.IO.FileSystem.Primitives" version="4.0.1" />
+        <dependency id="System.Linq.Expressions" version="4.0.11" />
+        <dependency id="System.Reflection" version="4.1.0" />
+        <dependency id="System.Reflection.Emit" version="4.0.1" />
+        <dependency id="System.Reflection.Extensions" version="4.0.1" />
+        <dependency id="System.Reflection.TypeExtensions" version="4.1.0" />
+        <dependency id="System.Runtime.Loader" version="4.0.0" />
+        <dependency id="Castle.Core" version="4.0.0-beta001" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="..\FakeItEasy\bin\Release\FakeItEasy.dll" target="lib\net40" />
+    <file src="..\FakeItEasy\bin\Release\FakeItEasy.pdb" target="lib\net40" />
+    <file src="..\FakeItEasy\bin\Release\FakeItEasy.xml" target="lib\net40" />
+    <file src="bin\Release\FakeItEasy.dll" target="lib\netstandard1.5" />
+    <file src="bin\Release\FakeItEasy.pdb" target="lib\netstandard1.5" />
+    <file src="bin\Release\FakeItEasy.xml" target="lib\netstandard1.5" />

```

Comment:
Ah I think we need to target 1.6 because of the `Microsoft.Extensions.DependencyModel` dependency.  I forgot to update the nuspec file.